packages/components/i18n/formatter.js

```diff
@@ -0,0 +1,7 @@
+exports.format = (messages) => {
+  const results = {};
+  Object.entries(messages).forEach(([id, msg]) => {
+    results[id] = msg.defaultMessage;
+  });
+  return results;
```

Comment:
Is this the same as:
```suggestion
  return Object.fromEntries(
    Object.entries(messages).map(([id, { defaultMessage }]) => ([id, defaultMessage])),
  );
```